### DISCUSSION TITLE
Custom UserStorageProvider properties and sync settings not available in new admin console

### DISCUSSION THREAD

**Discussion by benhag** [keycloak/keycloak#13718] [Archive]
We have implemented our own configurable UserStorageProvider. In the old Admin Console, the options for the provider were available. 
In the new Admin Console, neither our own options nor the settings for synchronisation are displayed. 

Should this work in the new Admin Console in the same way as in the old version or do we need to extend the theme for this? I can't find anything about this in the documentation.

**Comment by weltonrodrigo** [comment]
Issue https://github.com/keycloak/keycloak-admin-ui/issues/3059
